### ISSUE TITLE
"back to search" at this point seems wrong

### ISSUE THREAD

**Issue by tvb** [esotalk/esoTalk#368]
When you enter esotalk conversation list it basically acts as an index. After you have opened an conversation for reading and want to go back to the conversation list the tooltip is stating "back to search" Although I understand the reason behind it (the conversation list actually a search of all conversation) I think it should at this point say "back to index" instead.

![screen shot 2014-10-25 at 07 57 12](https://cloud.githubusercontent.com/assets/1263414/4778932/95c03b84-5c0c-11e4-8ceb-f3acd6fd04b6.png)

**Comment by jgknight** [comment]
I think a better compromise would be to change it to just say Back.  It is technically a search.  For example, go into the Development channel and search for esoTalk. Then click back, you'll still be on that same search.  I'm not sure that "index" is a better term.

**Comment by knownsyntax** [comment]
I don't think that this is an enhancement, more of a bug with the current system in which the back button loads. If you access the discussion board through a direct link to a thread, then anything but that thread including the index page will result in the back button loading up and thinking that thread is the default index page.
